manager/assets/modext/widgets/security/modx.panel.groups.roles.js

```diff
@@ -69,6 +80,7 @@ Ext.extend(MODx.panel.GroupsRoles,MODx.FormPanel,{
                             ,useSplitTips: true
                             ,monitorResize: true
                             ,width: 270
+                            ,minWidth: 270
                             ,minSize: 270
                             ,maxSize: 400
```

Comment:
Agreed but it feels like a separate issue to the bug fix. Maybe split this into a new issue?